cilium-health/launch/endpoint.go

```diff
@@ -120,7 +121,12 @@ func configureHealthInterface(netNS ns.NetNS, ifName string, ip4Addr, ip6Addr *n
 			return err
 		}
 
-		if ip6Addr != nil {
+		if ip6Addr == nil {
+			name := fmt.Sprintf("net.ipv6.conf.%s.disable_ipv6", ifName)
```

Comment:
Why not use `sysctl.Enable(fmt.Sprintf("net.ipv6.conf.%s.disable_ipv6", ifName))`? :-(

Reply:
Enable the disabling of the option? I figured it's easier to understand this way.

Reply:
We do have the helper functions to do this. It's not our fault the option is called `disable_ipv6` :-/ It's up to you.

Reply:
This is the reason I always try to avoid naming cilium options with `disable-foo` ;)

I recognize there was another option, but deliberately opted to use this call to reduce cognitive overhead for any future developers reading this code and attempting to understand it. Also, the value of 0/1 that we write to the sysctl will not change, it's stable kernel API at this point.